Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -812,246 +991,18 @@ namespace OpenInfraPlatform {
 					}
 				}
 
-				/*! \internal Still to refactor */

```

Comment:
These functions should stay in the file. (they were moved from RepresentationConverter).

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/efbf70dea68a1a215324e3c86caf18e571e17a13